recipes/libmagic/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{%set name = "libmagic" %}
+{%set version = "5.30" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "9cb61c1399eea2633688e0c2187b67f695210074fcbce2cd1f0701a2792f50dd" %}
+
+package:
+  name: libmagic
+  version: {{ version }}
+
+source:
+  fn: FILE{{ version.replace(".", "_") }}.tar.gz
+  url: https://github.com/file/file/archive/FILE{{ version.replace(".", "_") }}.tar.gz
```

Comment:
Any chance there's a more stable place to get this from? Oddly, the shas from github are not guaranteed to be stable over time. We've seen this issue in a number of feedstocks. [Here's one example](https://github.com/conda-forge/pandas-feedstock/pull/19#issuecomment-276019321). It's definitely ok to pull the archive from github though. Let me know your preference.